boards/arm/nucleo_l4r5zi/nucleo_l4r5zi_defconfig

```diff
@@ -55,3 +55,6 @@ CONFIG_UART_CONSOLE=y
 
 # Enable MPU
 CONFIG_ARM_MPU=y
+
+# Enable LPTimer
+CONFIG_STM32_LPTIM_TIMER=y
```

Comment:
LPTIM should not be enabled by default.
Rather, it should be enabled as new configuration in timer test suites